version.json

```
@@ -4,7 +4,7 @@
     "annotated git tag without the leading 'v'.",
     "This value gets updated automatically by `scripts/prepare-release`."
   ],
-  "vast-version-fallback": "2.4.0-rc2",
+  "vast-version-fallback": "2.4-rc3",
```

```suggestion
  "vast-version-fallback": "2.4.0-rc3",
```

I think you need to re-run the script with the missing patch version, not sure in what ways CMake will fail otherwise.

---

Right, I've opened a fixed PR